pkgs/development/libraries/nghttp2/default.nix

```diff
@@ -6,6 +6,7 @@
 , enableAsioLib ? false, boost ? null
 , enableGetAssets ? false, libxml2 ? null
 , enableJemalloc ? false, jemalloc ? null
+, enableApp ? false
```

Comment:
You could alternatively put the apps into their own output. Adding bin to outputs might suffice.

Reply:
Actually, this made $out basically empty, as libs are in $lib, so it's maybe a little "oversplit".  ($out isn't normally referenced)